elastic/kibana
# [RAM] Add missing privilege to alerting read operations

Closes #158957

## Summary

Adds the missing `getActionErrorLog` privilege. With the updated privileges, users with a custom Role including full access to  "Actions and Connectors", "Rule Settings" and "Stack Rules" can successfully inspect errored actions' logs:

![Errored actions logs](https://github.com/elastic/kibana/assets/18363145/0d34f6a3-d586-4fe7-b987-a829de0d852d)

## To Test

- Create a Role with `All` privileges granted in `Actions and Connectors`, `Rules Settings`, `Stack Rules` (under Kibana > Management) and assign it to a user
- Log in with that user
- Create a rule with a failing action (i.e. an Email Connector with wrong addresses)
- Wait for the rule to execute (or execute it manually)
- In the rule page, under `History` click the number under `Errored actions` in one of the rows of the logs table
- Check that error logs are visible in the flyout
